src/client/connstring.rs

```diff
@@ -22,7 +21,7 @@ impl Host {
         }
     }
 
-    fn new_ipc(ipc: String) -> Host {
+    fn with_ipc(ipc: String) -> Host {
```

Comment:
Actually, should be ```with_*```. std::vec::Vec provides a [```with_capacity``` constructor](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.with_capacity).